docs/reference_dataset/template_repository/hello-world-root-depls/cf-apps-deployments/generic-app/template/generic-app_manifest-tpl.yml

```diff
@@ -2,3 +2,19 @@ name: generic-app
 instances: (( grab secrets.generic-app.instances ))
 path: .
 
+memory: (( grab secrets.undefined_value || undefined.fallback.value || "64M" )) # as no value are defined, spruce will use the 64M default value
+
+# By default, spruce home dir is deployment directory (ie: credentials-resource/hello-world-root-depls/cf-apps-deployments/generic-app/)
+disk_quota: (( file "spruce-file-sample-from-secrets.txt" ))
+
+# To access generated files: "../../../../generated-files"
+# To access paas-templates directory, it is possible to use one of the following:
+#   - direct reference, quite similar, like  "../../../../additional-resource/hello-world-root-depls/cf-apps-deployments/generic-app/template"
+#   - use a variable see sample with timeout and timeout_from_file $CUSTOM_SCRIPT_DIR: (( grab $CUSTOM_SCRIPT_DIR )), in this sample, CUSTOM_SCRIPT_DIR=additional-resource/hello-world-root-depls/cf-apps-deployments/generic-app/template
+secrets: # this key is pruned by spruce
```

Comment:
`secrets: # you may define temporary keys in this section, that will be pruned upon spruce invocation, ie won't endup in spruce-generated files `